src/shared/group-record.c

```diff
@@ -372,3 +372,15 @@ bool group_record_match(GroupRecord *h, const UserDBMatch *match) {
 
         return true;
 }
+
+int group_record_is_root(const GroupRecord *g) {
```

Comment:
why int? I think we should correct the ret type of user_record_*, not the other way around

Reply:
I definitely didn't copy paste this from user-record.c without a second thought ...